prime-router/metadata/fhir_mapping/hl7/datatypes/NDL/Location.yml

```diff
@@ -0,0 +1,95 @@
+resourceType: Location
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+identifier_1:
+  condition: $hd1 NOT_NULL
+  vars:
+    hd1: String, HD.1
+  expressionType: nested
+  generateList: true
+  expressionsMap:
+    value_1:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: HD.1
+    extension_1-namespaceId:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: identifier-namespace-id
+        valueBoolean:
+          valueOf: "true"
+
+identifier_2:
+  condition: $hd2 NOT_NULL
+  expressionType: nested
+  generateList: true
+  vars:
+    hd2: String, HD.2
+  expressionsMap:
+    value:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: HD.2
+    type:
+      valueOf: datatype/CodeableConcept_var
+      generateList: true
+      expressionType: resource
+      vars:
+        code: HD.3
+      constants:
+        system: "http://terminology.hl7.org/CodeSystem/v2-0301"
+    extension_1-universalId:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: identifier-universal-id
+        valueBoolean:
+          valueOf: "true"
+
+identifier_3:
+  generateList: true
+  expressionType: nested
+  vars:
+    hd1: HD.1
+  condition: $hd1 NULL
+  expressionsMap:
+    value:
+      type: STRING
+      valueOf: $BASE_VALUE
+      expressionType: HL7Spec
+
+status:
+  valueOf: $ndlLocationStatus
+  type: STRING
+  expressionType: HL7Spec
+
+mode:
+  valueOf: $mode
+  type: STRING
+
```

Comment:
Looks like `mode` is should always be "instance" so can it just be set here rather than passed in?